component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticatorConstants.java

```diff
@@ -100,6 +100,10 @@ private TOTPAuthenticatorConstants() {
 
 	public static final String MAX_TOTP_ATTEMPTS_EXCEEDED = "MAX_TOTP_ATTEMPTS_EXCEEDED";
 
+	public static final String CONF_SHOW_AUTH_FAILURE_REASON_ON_LOGIN_PAGE = "showAuthFailureReasonOnLoginPage";
```

Comment:
`showAuthFailureReason` is the config used by the basic authenticator and SMS OTP authenticator. Shall we keep the config name consistent if the same behavior is expected?

Reply:
As per https://github.com/wso2-extensions/identity-local-auth-basicauth/blob/b95749214047d78d759f53ff364330d44ffc5d84/components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java#L427, `showAuthFailureReasonOnLoginPage` is what actually used to decide on sending the params required.
Should I change this ?